public/Word_dictionary/app.js

```diff
@@ -0,0 +1,121 @@
+let input = document.querySelector('#input');
+let searchBtn = document.querySelector('#search');
+let apiKey = '771aa8f7-d363-4ff0-a824-10181a86ac47';
```

Comment:
remove this key it makes website dangerous

Reply:
i have done